hermes_base/settings.py

```diff
@@ -240,20 +255,16 @@
             # Group ID must be prefixed with SCiMMA SCRAM credential username to open the SCiMMA kafka stream
             'GROUP_ID': os.getenv('SCIMMA_AUTH_USERNAME', '') + '-' + os.getenv('HOPSKOTCH_GROUP_ID', 'hermes-dev'),
             'TOPIC_HANDLERS': {
-                'hermes.test': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
-                'hermes.astrometry': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
-                'hermes.discovery': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
-                'hermes.message': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
-                'hermes.photometry': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
-                'hermes.spectroscopy': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
+                '*': 'hermes.alertstream_handlers.ingest_from_hop.handle_generic_message',
+                'hermes.*': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
                 'tomtoolkit.test': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
                 'gcn.circular': 'hermes.alertstream_handlers.ingest_from_hop.handle_gcn_circular_message',
-                #'*': 'hermes.alertstream_handlers.ingest_from_hop.handle_generic_message',
+                'igwn.gwalert': 'hermes.alertstream_handlers.ingest_from_hop.handle_igwn_message'
             },
         },
     },
     {
-        'ACTIVE': True,
+        'ACTIVE': False,
```

Comment:
Right now there is nothing to ingest from GCN classic over kafka. We still want to ingest LVC_COUNTERPART right now, but they don't have that stream setup yet I think. I believe this is analogous to the `gcn.notice` topic in hop, but that is in VOEvent format (vs much simpler plain text in GCN classic) so I am hoping hop can either have a better format since it is trying to have everything in avro/json, not VOEvent. If we need to though we can try to write a parser for VOEvent. I would probably just want to convert the whole message out of VOEvent and into json to store, since itll be much easier to read and parse.